File: Matrices/Easy/1572/1572.c
int diagonalSum(int** mat, int matSize, int* matColSize){
    int sum = 0;
    for(int i = 0; i < matSize; i++){
        sum += mat[i][i];
        sum += mat[matSize - i - 1][i];
    }
    if(matSize % 2){
        sum -= mat[matSize/2][matSize/2];
    }
    return sum;
}

File: Matrices/Easy/1572/1572.py
from typing import List

class Solution:
    def diagonalSum(self, mat: List[List[int]]) -> int:
        sum = 0
        for i in range(len(mat)):
            sum += mat[i][i]
            sum += mat[len(mat)-i-1][i]
        if len(mat) % 2:
            sum -= mat[len(mat)//2][len(mat)//2]
        return sum